test/lib/ufe/testGroupCmd.py

```diff
@@ -109,6 +109,12 @@ def testUsdGroup(self):
             ufeSelectionList, newGroupName)
         groupCmd.execute()
 
+        # Group object (a.k.a parent) will be added to selection list. This behavior matches the native Maya group command.
+        globalSelection = ufe.GlobalSelection.get()
+
+        groupPath = ufe.Path([mayaPathSegment, usdUtils.createUfePathSegment("/Ball_set/Props/newGroup1")])
+        self.assertEqual(globalSelection.front(), ufe.Hierarchy.createItem(groupPath))
+
```

Comment:
test driven development ( TDD ). Wrote the test first, it failed. Added new code logic and pass.